google-cloud-tools-plugin/testSrc/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleDeploymentEditorTest.java

```diff
@@ -389,13 +389,12 @@ public void testDockerfileOverride() {
     assertEquals("an override", editor.getDockerfileTextField().getText());
   }
 
-  public void testDockerfileStartsDisabledAndWithModuleSetting() {
+  public void testDockerfileStartsDisabled() {
```

Comment:
I need to verify this. Specifically checking to see if this unit test was valid and why my changes changed the behavior. @joaoandremartins if anything jumps out at you let me know since you wrote this originally.